<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TerminalFrame</class>
 <widget class="QFrame" name="TerminalFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTextEdit" name="textEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="comboBox_address"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_message"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_transmit">
       <property name="text">
        <string>Отправить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
